e2e/src/tests/entityIcon.test.ts

```diff
@@ -40,19 +40,18 @@ test(
         )
         .toBe(allExpectedEntities.length);
 
-      for (const entity of allExpectedEntities) {
-        const actualEntity = actualEntitiesIcons.find(
-          (e) => e.entityName === entity.name,
-        )!;
-        const expectedEntityIcon = await apiHelper.getEntityIcon(entity);
-        expect
-          .soft(
-            actualEntity.icon,
-            `${ExpectedMessages.entityIconIsValid} for ${entity.name}`,
-          )
-          .toBe(expectedEntityIcon);
-      }
-
+      const randomEntity =
+        GeneratorUtil.randomArrayElement(allExpectedEntities);
```

Comment:
what do you mean by 'we can have random at results of test'? Here I simply choose one random model/app to verify its icon instead of verifying all model/app icons (this is slow-down test execution and the number of models is growing every release). Using random values in testing helps to identify hidden issues and increases test coverage